content/10.diagnostics.md

```diff
@@ -28,6 +28,27 @@ In this case the test was designed to detect sarbecoviruses, of which SARS-CoV-2
 
+##### dPCR
+
+Digital PCR (dPCR) is a new generation of PCR technologies where a sample is partitioned into thousands of compartments, such as nanodroplets or nanowells, and a PCR reaction takes place in each compartment.
+This leads to a digital read-out where each partition is either positive or negative for the nucleic acid sequence being tested for.
+While dPCR equipment is not as common yet as that for RT-PCR, dPCR generally has higher sensitivity than can be achieved with other PCR technologies, while maintaining high specificity [@doi:10.3390/s18041271].
+High sensitivity is particularly relevant for SARS-CoV-2 detection, since low viral load in clinical samples can lead to false negatives.
+Suo et al. performed a double-blind evaluation of droplet dPCR (ddPCR) for SARS-CoV-2 detection on 57 samples--43 samples from suspected positive patients, and 14 from supposed convalescents--that had all tested negative for SARS-CoV-2 using RT-PCR.
+Despite the initial negative results, 33 out of 35 (94.3%) patients were later clinically confirmed positive.
+All of these individuals tested positive using ddPCR.
+Additionally, of 14 supposed convalescents who had received two consecutive negative RT-PCR tests, nine (64.2%) tested positive for SARS-CoV-2 using ddPCR. 
+Two symptomatic patients tested negative with both RT-PCR and ddPCR, but were later clinically diagnosed positive, and 5 of the 14 suspected convalescents tested negative by ddPCR [@doi:10.1101/2020.02.29.20029439].
+While not a complete head-to-head comparison to RT-PCR in all aspects-- e.g. no samples testing positive using RT-PCR were evaluated by ddPCR-- the study shows the potential of dPCR for viral detection in highly diluted samples.
+In a second study, Dong et al. [@doi:10.1101/2020.03.14.20036129] compared the results of qRT-PCR and ddPCR testing for SARS-CoV-2 in 194 samples, including 103 samples from suspected patients, 75 from contacts and close contacts, and 16 from suspected convalescents.
+Of the 103 suspected patient samples, 29 were reported as positive, 25 as negative, and 49 as suspected by qRT-PCR; all patients were later confirmed to be SARS-CoV-2 positive.
+Of the qRT-PCR negative or suspected samples, a total of 61 (17 negative and 44 suspected) were later confirmed to be positive by ddPCR, improving the overall detection rate among these patients from 28.2% to 87.4%.
+Of 75 patient samples from contacts and close contacts, 48 were tested negative with both methods and patients indeed remained healthy.
+Within the remaining 27 patient samples, 10 tested positive, 1 negative, 16 suspect by qRT-PCR.
+15 out of 16 suspect samples and the negative test result were overturned by ddPCR, decreasing the rate of suspect cases from 21% to 1%.
+Importantly, all samples tested positive using qRT-PCR were also tested positive using ddPCR.
+Overall these studies suggest that dPCR is a promising tool for overcoming the problem of false negative tests in SARS-CoV-2 detection.
```

Comment:
* "Overall, these studies"
* "false-negative"
* Testing and detection sound repetitive: "the problem of false-negative SARS-CoV-2 testing"